pipeline/change_detect.go

```diff
@@ -0,0 +1,75 @@
+package pipeline
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// Compute the changeDetect of a stream or batch.
+// The changeDetect is computed on a single field,
+// discarding consecutive duplicate values, detecting
+// detects the points at which the series field
+// changes from one value to another.
```

Comment:
I think it would be good to mention the output of the changeDetect node, i.e. that it just passes the data through unmodified and therefore acts like a special kind of filter.

Reply:
FYI, the Kapacitor docs [here](https://docs.influxdata.com/kapacitor/v1.4/nodes/) are autogenerated from these comments on the types in the pipeline package.